### PR TITLE
Fix bug in export using FQ ONNX in replacing activation holder with more than 8 bits with identity

### PR DESCRIPTION
**Pull request by lapid92**
## Checklist before requesting a review:
- [ ] I set the appropriate labels on the pull request.
- [ ] I have added/updated the release note draft (if necessary).
- [ ] I have updated the documentation to reflect my changes (if necessary).
- [ ] All function and files are well documented. 
- [ ] All function and classes have type hints.
- [ ] There is a licenses in all file.
- [ ] The function and variable names are informative. 
- [ ] I have checked for code duplications.
- [ ] I have added new unittest (if necessary).

### CODE REVIEW

File: model_compression_toolkit/exporter/model_exporter/pytorch/fakely_quant_onnx_pytorch_exporter.py [hunk 1]
```diff
@@ -78,10 +78,17 @@ def export(self, output_names=None) -> None:
...
+            for act_holder in act_holder_list: # pragma: no cover
+                obj = self.model
+                attrs = act_holder.split(".")
+                for a in attrs[:-1]:
+                    obj = getattr(obj, a)
+                if hasattr(obj, attrs[-1]):
+                    delattr(obj, attrs[-1])
+                    setattr(obj, attrs[-1], torch.nn.Identity())
+                else:
+                    Logger.info(f"Failed to delete activation quantization holder {act_holder}, "
```
Comment (elad-c) [inline model_compression_toolkit/exporter/model_exporter/pytorch/fakely_quant_onnx_pytorch_exporter.py]: Make the message more informative. Something like: "During removal of activation quantization of quantizer with more than 8 bits, deleting act blala failed, ...."

Comment (ofirgo) [reply model_compression_toolkit/exporter/model_exporter/pytorch/fakely_quant_onnx_pytorch_exporter.py:90]: You don't verify that it is more than 8 bits, do you?
Do we want to delete it only if it has more than 8bit?
